src/main/java/io/jenkins/plugins/pipeline_cloudwatch_logs/CloudWatchAwsGlobalConfiguration.java

```diff
@@ -142,7 +142,7 @@ FormValidation validate(String logGroupName, String region, String credentialsId
             // TODO should also check DescribeLogStreams, and perhaps even CreateLogStream and PutLogEvents, to ensure roles are correct
         } catch (Exception x) {
             String msg = processExceptionMessage(x);
-            return FormValidation.error(StringUtils.abbreviate(msg, 200));
+            return FormValidation.error(abbreviate ? StringUtils.abbreviate(msg, 200) : msg);
```

Comment:
I think this was just missed in https://github.com/jenkinsci/pipeline-cloudwatch-logs-plugin/pull/64. I needed the full message to debug some errors while I was configuring things for the tests.